sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -159,3 +162,244 @@ export function toBuffer(input: any): Buffer {
   log.utils("[utils.toBuffer] The converted buffer is: %O.", result);
   return result;
 }
+/**
+ * @internal
+ * Helper function to retrieve active receiver name, if it exists.
+ */
+export function getAssociatedReceiverName(
```

Comment:
Is there some kind of merge issue? 
This function should not be appearing in the diffs for this PR

Reply:
Fixed